### DISCUSSION TITLE
Unreal puerts中怎么添加runtime下的一些模块，比如PakFile?

### DISCUSSION THREAD

**Discussion by Shirlman** [Tencent/puerts#1919] [Q&A]
希望在ue.d.ts中包含指定的runtime模块，比如引入PakFile，那PakFile模块中的定义会自动生成到ue.d.ts中并可以在typescript中调用

**Comment by chexiongsheng** [comment]
看文档，文档有，不仅ue api，任意第三方api都能声明使用。
其次看你这描述，你对ue.d.ts的作用有些误解。
